linode_api4/linode_client.py

```diff
@@ -1433,6 +1433,620 @@ def cancel(self):
         self.client.post("/object-storage/cancel", data={})
         return True
 
+    def transfer(self):
+        """
+        The amount of outbound data transfer used by your account’s Object Storage buckets,
+        in bytes, for the current month’s billing cycle. Object Storage adds 1 terabyte
+        of outbound data transfer to your data transfer pool.
+
+        API Documentation: https://www.linode.com/docs/api/object-storage/#object-storage-transfer-view
+
+        :returns: The amount of outbound data transfer used by your account’s Object
+                  Storage buckets, in bytes, for the current month’s billing cycle.
+        :rtype: integer
+        """
+        result = self.client.get("/object-storage/transfer")
+
+        if not "used" in result:
+            raise UnexpectedResponseError(
+                "Unexpected response when getting Transfer Pool!"
```

Comment:
I think it may be nicer to pass the response body in so the user may have a better idea about what went wrong.
Similar to this:
https://github.com/linode/linode_api4-python/blob/1456769ce88c5fc1e963dd924bdf5fc8a1f1c7c0/linode_api4/objects/linode.py#L76